test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -456,6 +457,8 @@ def setUp(self):
         # setup block, rest of them to be added to this block
         # once save & load configurations are implemented for
         # comm, mom, scheduler
+        if self.use_cur_setup:
```

Comment:
I think you should move the

        if self.use_cur_setup:
            self.server.delete_current_state()
            self.server.load_configuration(self.saved_file)

block here from tearDown() instead of adding this.

Reply:
I tried to move this block from teardown to setup() but i was still facing the same problem.
So keeping this change